core/storage/statistics/gae_models.py

```diff
@@ -61,7 +61,7 @@
 # mapping is useful to uniquely identify issues by the combination of their
 # issue type and other type-specific information (such as the list of states
 # involved).
-ISSUE_TYPE_KEYNAME_MAPPING = {
+ISSUE_IDENTIFIYING_CUSTOMIZATION_ARG = {
```

Comment:
I don't really understand this name TBH. Do you mean something like ISSUE_TYPE_TO_KEY_WITH_STATE_INFO?

Also note that IDENTIFYING is misspelled.